src/main/app/forms/validation/validators/isCountrySupported.ts

```diff
@@ -13,13 +13,22 @@ import { request } from 'client/request'
 
 const postcodeClient = new PostcodeInfoClient(config.get<string>('postcodeLookup.apiKey'), request)
 
+export enum BlockedPostcodes {
```

Comment:
Do not export it, it is internal implementation until we have a reason to make it public.